fab/inventory/icds-new

```diff
@@ -304,8 +304,8 @@ hostname='couch1'
 10.247.164.14
 
 [couch2:vars]
-devices=["/dev/sdb"]
-partitions=["/dev/sdb1"]
+devices=["/dev/sdb","/dev/sdb"]
```

Comment:
Is this supposed to say the same thing twice? Or is this supposed to say `/dev/sdc` with a c?

Reply:
(pretty unfamiliar with our LVM setup conventions, so just wanted to double check)